frontend/packages/webterminal-plugin/src/cloud-shell/CloudShell.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
+import { FLAG_DEVWORKSPACE } from '@console/app/src/consts';
```

Comment:
This flag is only used in the `webterminal-plugin`

Can you move `FLAG_DEVWORKSPACE` and `FLAG_V1ALPHA2DEVWORKSPACE` also from

`console-app/src/consts.ts` into
`webterminal-plugin/src/consts.ts`?